packages/subgraph/manifest/subgraph.placeholder.yaml

```diff
@@ -64,15 +64,15 @@ dataSources:
         - PluginGrantedPermission
       abis:        
         - name: PluginSetupProcessor
-          file: $ZARAGOZA_CONTRACTS_MODULE/artifacts/contracts/plugin/PluginSetupProcessor.sol/PluginSetupProcessor.json
+          file: $ZARAGOZA_CONTRACTS_MODULE/artifacts/contracts/framework/plugin/setup/PluginSetupProcessor.sol/PluginSetupProcessor.json
```

Comment:
Can we rename `ZARAGOZA_CONTRACTS_MODULE` to just `CONTRACTS_MODULE` or `OS_CONTRACTS_MODULE`?

Reply:
This is probably a job for @mathewmeconry